consensus/istanbul/backend/announce.go

```diff
@@ -37,6 +38,10 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
+const (
+	timestampDiffThreshold = 20 * time.Second
```

Comment:
thinking it would be ok if this was a bit bigger? what would be the effect of making this the same as the announce interval?

Reply:
I don't see any major issues with making this bigger.